### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by mryeminthu**
- Use preprocessed HTML file with embedded Ruby code.
- Use layouts and templates for shared content.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  before_action :set_post
+
+  def create
+    @comment = @post.comments.build(comment_params.merge(user: @current_user))
+
+    if @comment.save
```
Comment (Shedrack-Sunday) [inline app/controllers/comments_controller.rb]: - [ ] A comment is created on behalf of the currently logged in user (current_user) of the app.
- [ ] Kindly note that this  comments controller should have new method also 
- [ ] Please make sure it has a route also which will be a `GET`.

### Sample:
`GET /posts/:post_id/comments/new(.:format) comments#new`

### Sample 
```ruby

  def new
    @comment = Comment.new
  end
```

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @mryeminthu      👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with Rails features ✔️
- Great work on the  API routes  implementation using 👏
- Kudus on using partials 👏    

### Confirmed view:

image


# - Required Changes  🌵  

- [ ] Kindly note  the comment i made regarding the missing views and routes to be added for a method in the comment controller 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Shedrack-Sunday** [comment]
Hello @mryeminthu ,

I mistakenly approved your work instead of asking for changes. Kindly make the requested change while the team rectify this error from my end on your dashboard. My sincere apologies 👍 

Thank you

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (mryeminthu) [reply app/controllers/comments_controller.rb]: @Shedrack-Sunday 
Hello Sir, I have already done as you mentioned. Could I submit my project now?

Comment (mryeminthu) [reply app/controllers/comments_controller.rb]: I am ready to submit. And I am waiting the response.

Comment (mryeminthu) [reply app/controllers/comments_controller.rb]: Dear  Mr. @Shedrack-Sunday,
I waited for about 4 hours to rectify my dashboard, but nothing changed, so I submitted the project.

**Review by Nemwel-Boniface — COMMENTED** [review]
# Hello @mryeminthu  :wave: 

Thank you for making the changes that were requested by the previous reviewer @Shedrack-Sunday . I want to let you know that your project is approved and you are free to merge this PR and work on other milestones (If you have not yet continued to work on them please make sure that you do so).

As always #happycoding! :rocket:

**Pull request merged**